### DISCUSSION TITLE
Question about getting a standalone Tailscale LXC

### DISCUSSION THREAD

**Discussion by bobloadmire** [tteck/Proxmox#3290] [General]
For some reason you cant use the tailscale script to generate a Tailscale LXC. not sure why, I just want to have a Tailscale LXC since it will be my exit node, and functioning as my VPN server and I can access all of the open ports from other services I have installed on proxmox from there. Am I misunderstanding how this is supposed to be used? I thought my use case would be common. It doesn't make sense to me just to install it on an arbitrary LXC I already have installed, i'm going to forget where I installed it, or make a breaking change not realizing I had Tailscale on that particular arbitrary LXC. Thanks.

**Comment by tteck** [comment]
You want Headscale then

**Reply by bobloadmire** [reply]
lmao TIL Headscale is a thing! thanks

**Reply by bobloadmire** [reply]
oh you might want to put Tailscale as a search keyword for it, when I type Tailscale into the website, headscale doesn't come up, so I might have seen it otherwise.
